script.js

```diff
@@ -0,0 +1,150 @@
+const toDoForm = document.querySelector(".toDoForm");
+const toDoInput = toDoForm.querySelector("input");
+const toDoList = document.querySelector(".toDoList");
+const finishedList = document.querySelector(".finishedList");
+const lenTodo = document.getElementById("todo-count");
+const lenFin = document.getElementById("fin-count");
+
+const TODODATA = "todoData"
+const FINDATA = "FinData"
+
+let toDoArr = [];
+let finArr = [];
+
+function loadDATA(){
+    const loadedTODO = localStorage.getItem(TODODATA);
+    const loadedFIN = localStorage.getItem(FINDATA);
+    
+    if (loadedTODO !== null) {
```

Comment:
```suggestion
    if (loadedToDo) {
```
이렇게 간단히 줄여볼 수 있을거 같네요!